src/Dapr.Actors/Runtime/ActorManager.cs

```diff
@@ -340,6 +339,13 @@ private async Task<T> DispatchInternalAsync<T>(ActorId actorId, ActorMethodConte
             T retval;
             try
             {
+                // Set the state context of the request, if possible.
+                var stateManager = state.Actor.StateManager;
+                if (typeof(IActorContextualState).IsAssignableFrom(stateManager.GetType()))
+                {
+                    await (stateManager as IActorContextualState).SetStateContext(Helper.GetCallContext());
```

Comment:
`Helper.GetCallContext()` - I want to see a different name for this. `Helper` or `Util` is a solid clue that this should be factored differently. Will make a suggestion there.